.github/workflows/ci.helm-lint.yaml

```diff
@@ -0,0 +1,48 @@
+name: CI Helm Lint
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+      - gcp-cypress-prod
```

Comment:
I think we want to run it only on `gcp-baobab-prod` 🤔

Reply:
Also I think so, thank you for suggested. 🙏